include/zephyr/linker/devicetree_regions.h

```diff
@@ -187,6 +205,30 @@
 #define _CHECK_APPLY_FN(compat, EXPAND_MPU_FN, ...)					\
 	DT_FOREACH_STATUS_OKAY_VARGS(compat, _CHECK_ATTR_FN, EXPAND_MPU_FN, __VA_ARGS__)
 
+/**
+ * Check that the node_id has both properties:
+ *  - zephyr,memory-region-mmu
+ *  - zephyr,memory-region
+ *  - reg
+ *
+ * and call the EXPAND_MMU_FN() macro
+ */
+#define _CHECK_MMU_ATTR_FN(node_id, EXPAND_MMU_FN, ...)						\
```

Comment:
Can you rename the current `_CHECK_ATTR_FN` to `_CHECK_MPU_ATTR_FN` for coherence as well?

Reply:
Done.